<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Проверка теоретической модели</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,0,2">
          <property name="spacing">
           <number>8</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="leftLayout" stretch="0,0">
            <property name="spacing">
             <number>8</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>8</number>
            </property>
            <property name="topMargin">
             <number>8</number>
            </property>
            <property name="rightMargin">
             <number>8</number>
            </property>
            <property name="bottomMargin">
             <number>8</number>
            </property>
            <item>
             <widget class="QScrollArea" name="scrollArea">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_3">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>509</width>
                 <height>334</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="1" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_9" stretch="0">
                  <property name="spacing">
                   <number>4</number>
                  </property>
                  <item>
                   <layout class="QHBoxLayout" name="formula_hl_demo">
                    <property name="spacing">
                     <number>8</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label__demo">
                      <property name="font">
                       <font>
                        <family>Cambria Math</family>
                        <pointsize>14</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>F := </string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit_7">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>Cambria Math</family>
                        <pointsize>21</pointsize>
                       </font>
                      </property>
                      <property name="cursor">
                       <cursorShape>IBeamCursor</cursorShape>
                      </property>
                      <property name="clearButtonEnabled">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item row="0" column="0">
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="2" column="0">
                 <widget class="QPushButton" name="add_expression_pb">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="font">
                   <font>
                    <family>Cambria</family>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Добавить выражение</string>
                  </property>
                  <property name="checkable">
                   <bool>false</bool>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="evaluate_pb">
              <property name="font">
               <font>
                <family>Cambria</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Рассчитать</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="rightLayout" stretch="0,1">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="font">
               <font>
                <family>Cambria</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="text">
               <string>Результат:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignHCenter|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="answer_label">
              <property name="font">
               <font>
                <family>Cambria</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignHCenter|Qt::AlignTop</set>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Построение модели по выборке</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,0,1">
          <property name="spacing">
           <number>8</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="leftLayout_3" stretch="0,0,1,0,0">
            <property name="spacing">
             <number>8</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>8</number>
            </property>
            <property name="topMargin">
             <number>8</number>
            </property>
            <property name="rightMargin">
             <number>8</number>
            </property>
            <property name="bottomMargin">
             <number>8</number>
            </property>
            <item>
             <widget class="QScrollArea" name="scrollArea_3">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_5">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>628</width>
                 <height>67</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="1" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_11" stretch="0,0">
                  <property name="spacing">
                   <number>4</number>
                  </property>
                  <item>
                   <layout class="QHBoxLayout" name="formula_hl_demo_4">
                    <property name="spacing">
                     <number>8</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label__demo_4">
                      <property name="font">
                       <font>
                        <family>Cambria Math</family>
                        <pointsize>14</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Выр := </string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit_10">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>Cambria Math</family>
                        <pointsize>21</pointsize>
                       </font>
                      </property>
                      <property name="cursor">
                       <cursorShape>IBeamCursor</cursorShape>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                      <property name="placeholderText">
                       <string>v = a0 *sqrt(N) + a1 * D^2 + a2 / L</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
                    <item>
                     <widget class="QLabel" name="label__demo_5">
                      <property name="font">
                       <font>
                        <family>Consolas</family>
                        <pointsize>14</pointsize>
                        <weight>50</weight>
                        <italic>false</italic>
                        <bold>false</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>Пример:</string>
                      </property>
                      <property name="textFormat">
                       <enum>Qt::AutoText</enum>
                      </property>
                      <property name="textInteractionFlags">
                       <set>Qt::NoTextInteraction</set>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label__demo_6">
                      <property name="palette">
                       <palette>
                        <active>
                         <colorrole role="WindowText">
                          <brush brushstyle="SolidPattern">
                           <color alpha="255">
                            <red>125</red>
                            <green>125</green>
                            <blue>125</blue>
                           </color>
                          </brush>
                         </colorrole>
                        </active>
                        <inactive>
                         <colorrole role="WindowText">
                          <brush brushstyle="SolidPattern">
                           <color alpha="255">
                            <red>125</red>
                            <green>125</green>
                            <blue>125</blue>
                           </color>
                          </brush>
                         </colorrole>
                        </inactive>
                        <disabled>
                         <colorrole role="WindowText">
                          <brush brushstyle="SolidPattern">
                           <color alpha="255">
                            <red>120</red>
                            <green>120</green>
                            <blue>120</blue>
                           </color>
                          </brush>
                         </colorrole>
                        </disabled>
                       </palette>
                      </property>
                      <property name="font">
                       <font>
                        <family>Consolas</family>
                        <pointsize>14</pointsize>
                        <weight>50</weight>
                        <italic>false</italic>
                        <bold>false</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>v = a0 *sqrt(N) + a1 * D^2 + a2 / L</string>
                      </property>
                      <property name="textFormat">
                       <enum>Qt::AutoText</enum>
                      </property>
                      <property name="textInteractionFlags">
                       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Cambria</family>
               </font>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="text">
               <string>Выборка данных:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="textBrowser">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="formula_hl_demo_5">
              <property name="spacing">
               <number>8</number>
              </property>
              <item>
               <widget class="QLineEdit" name="lineEdit_11">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Consolas</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>IBeamCursor</cursorShape>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="placeholderText">
                 <string>C:/path/to/file</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="select_file_pb">
                <property name="font">
                 <font>
                  <family>Cambria</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Выбрать файл </string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="evaluate_pb_">
              <property name="font">
               <font>
                <family>Cambria</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Рассчитать коэффициенты линейной регрессии</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="rightLayout_3" stretch="0,1">
            <item>
             <widget class="QLabel" name="result_label_2">
              <property name="font">
               <font>
                <family>Cambria</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="text">
               <string>Результат:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignHCenter|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="result_label">
              <property name="font">
               <font>
                <family>Cambria</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignHCenter|Qt::AlignTop</set>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>929</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Загрузить формулы...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
